server/app.py

```diff
@@ -1,11 +1,116 @@
-from flask import Flask
+from flask import Flask, request, jsonify, url_for, redirect, session
+from config import DevelopmentConfig
+import os
+from datetime import timedelta
+
+# dotenv setup
+from dotenv import load_dotenv
+load_dotenv()
+
+# Routes
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
+from api.user_handler import user_handler
+from api.meeting_handler import meeting_handler
+from api.appointment_handler import appointment_handler
+from api.availability_handler import availability_handler
+
+#OAuth Library 
+from authlib.integrations.flask_client import OAuth
```

Comment:
Hi Michael, tried running the server but had an import error here on line 19 in app.py (flask.cli.NoAppException: While importing "app", an ImportError was raised:). Kindly confirm it ran okay. I merged with my branch on my system and wouldn't want to commit to my branch just yet.

Reply:
@DanielOttah did you run pipenv install? This needs the authlib library installed to work.